index.js

```diff
@@ -89,7 +89,7 @@ var listenerCount = EventEmitter.listenerCount ||
  * @public
  */
 
-function send (req, path, options) {
+function send(req, path, options) {
```

Comment:
Please remove all of these changes where you're only adding or removing white space as it only clutters the diff and adds nothing of value.